jdi-light/src/main/java/com/epam/jdi/light/actions/ActionHelper.java

```diff
@@ -343,7 +343,7 @@ public static String getBeforeLogString(JoinPoint jp) {
         return map;
     }
     public static void processPage(ActionObject jInfo) {
-        getWindows();
```

Comment:
please revert
we can't remove this here. this will fail jdi tests

Reply:
Test's works fine without this line